MVP/defaults-build.yaml

```diff
@@ -0,0 +1,43 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7'
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: target
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
+    wrappers:
+        - ansicolor
+        - workspace-cleanup
+        - timestamps
+        - build-name:
+            name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
```

Comment:
I think the only reason you have 'version' and 'release' as string parameters is because it can be set in the build name. Is this required for '_MVP_' ?